Resources/config/hydra.xml

```diff
@@ -23,9 +29,11 @@
         </service>
 
         <service id="api.hydra.listener.request_exception" class="Dunglas\ApiBundle\Hydra\EventListener\RequestExceptionListener">
-            <argument type="service" id="api.hydra.normalizer.error" />
+            <argument type="string">api.hydra.controller.exception:showAction</argument>
+            <argument type="service" id="logger" on-invalid="null" />
 
-            <tag name="kernel.event_listener" event="kernel.exception" method="onKernelException" />
```

Comment:
I would keep the `event_listener` tag instead of the subscriber. You can use the `priority` attribute to set the priority.